js/main.js

```diff
@@ -26,6 +28,18 @@ function storeValues(event) {
   placeHolder.setAttribute('src', './images/placeholder-image-square.jpg');
   viewSwap('entries');
   toggleNoEntries();
+
+  if (data.editing !== null) {
+    for (var i = 0; i < data.entries.length; i++) {
+      if (data.editing.entryId === data.entries[i].entryId) {
+        data.entries[0].entryId = data.entries[i].entryId;
```

Comment:
This is moving the edited entry to the beginning of the list, which is not the intended functionality. You should be replacing the original object for the entry at its original position in `data.entries`